lib/factor/ConstantFactors.js

```diff
@@ -43,8 +43,11 @@ ConstantFactors.getPrimeFactors = function(number){
 ConstantFactors.getFactorPairs = function(number){
   const factors = [];
 
-  const root = Math.sqrt(number);
-  for (var divisor = 1; divisor <= root; divisor++) {
+  const bound = Math.floor(Math.sqrt(Math.abs(number)));
+  for (var divisor = -bound; divisor <= bound; divisor++) {
```

Comment:
I actually pre-emptively added this function when adding other factoring functionality. Which caused me some confusion and which is why I think it's best to wait for the case to arise now hahaha